### ISSUE TITLE
Create and add user stories to the README

### ISSUE THREAD

**Issue by A-PS1999** [UoB-COMSM0110/2024-group-1#8]

**Comment by A-PS1999** [comment]
Following some staff feedback, it may be a good idea to amend some of the user stories we've come up with.

**Comment by A-PS1999** [comment]
Update: The pull request by Jasmine, namely #16 relates to this issue. Once that pull request is merged, this issue can be moved to 'done'.

**Comment by A-PS1999** [comment]
Now moving this to 'done' as #16 has been merged into `main`.
